### DISCUSSION TITLE
Need I put the utxo with runes as input to the rune transfer transaction?

### DISCUSSION THREAD

**Discussion by donut33-social** [ordinals/ord#3561] [Q&A]
I just learned how to transfer a rune with OP_RETURN. But I have some questions:
1. If I want to transfer some of my own rune to another address. Need I use the utxo which has the rune or any of my utxos?
2. If I have 10000 A rune, and I transfer 4000 A rune to another one, which utxo hold my left 6000 rune?
3. When I etching a new rune, the run name is in the commit transaction script, And what about the data of tag: rune in the runestone,if the tag rune data is mismatch with the name in ordinales script, what will happen?

**Comment by raphjaph** [comment]
Just like with transferring bitcoin you will have to transfer the left over runes to your change address
